src/gui/plugins/component_inspector/ComponentInspector.qml

```diff
@@ -399,11 +399,215 @@ Rectangle {
+            text: "Camera >"
+
+            MouseArea {
+              id: viewSubCameraArea
+              anchors.fill: parent
+              hoverEnabled: true
+              onEntered: cameraSubmenu.open()
+            }
+          }
+      
+          MenuItem {
+            id: contact
+            text: "Contact"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(contact.text, "sensor");
+            }
+          }
+      
+          MenuItem {
+            id: forceTorque
+            text: "Force torque"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(forceTorque.text, "sensor");
+            }
+          }
+      
+          MenuItem {
+            id: gps
+            text: "GPS"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(gps.text, "sensor");
+            }
+          }
+      
+          MenuItem {
+            id: imu
+            text: "IMU"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(imu.text, "sensor");
+            }
+          }
+      
+          MenuItem {
+            id: lidarSensorMenu
+            text: "Lidar >"
+
+            MouseArea {
+              id: viewSubLidarArea
+              anchors.fill: parent
+              hoverEnabled: true
+              onEntered: cameraSubmenu.open()
+            }
+          }
+     
+          MenuItem {
+            id: magnetometer
+            text: "magnetometer"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(magnetometer.text, "sensor");
+            }
+          }
+        }
+      
+        Menu {
+          id: cameraSubmenu
+          x: addSensorMenu.x - addSensorMenu.width
+          y: addSensorMenu.y + cameraSensorMenu.y
+
+          MenuItem {
+            id: depth
+            text: "Depth"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("depth_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: logical
+            text: "Logical"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("logical_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: monocular
+            text: "Monocular"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: multicamera
+            text: "Multicamera"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("multicamera", "sensor");
+            }
+          }
+          MenuItem {
+            id: rgbd
+            text: "RGBD"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("rgbd_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: thermal
+            text: "Thermal"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("thermal_camera", "sensor");
+            }
+          }
```

Comment:
add a `MenuItem` for `segmentation_camera`?

Reply:
Done